### ISSUE TITLE
Syntax highlighting is broken since yesterday

### ISSUE THREAD

**Issue by LeCoupa** [vuejs/vetur#1082]
Hello, I don't know if it's related to one of the recent updates but syntax highlighting is broken since yesterday. I am using pug.

<img width="595" alt="screenshot 2019-02-08 at 10 19 49" src="https://user-images.githubusercontent.com/1658644/52469194-1d16e900-2b8b-11e9-8b88-ea3093bc563a.png">

Here is the code if you want to try it out by yourself

```pug
<template lang="pug">
div(
  @mouseenter="showTooltip = true"
  @mouseleave="showTooltip = false"
  :class=`[
    "c-conversation-box-content-message-bubble",
    "c-conversation-box-content-message-bubble--" + from,
    "c-conversation-box-content-message-bubble--" + type
  ]`
)
</template>
```

**Comment by Patcher56** [comment]
Before the yesterday update 0.15.0 there was no syntax highlighting for vue in pug.
Now the content between the " " gets highlighted in javascript.

Because in your code you are using the multiline string with the `` the js can't be highlighted correctly.

For a workaround, please put it in one line:

```html
div(
  @mouseenter="console.log(true)"
  @mouseleave="console.log(false)"
  :class="['c-conversation-box-content-message-bubble', 'c-conversation-box-content-message-bubble--' + 'x', 'c-conversation-box-content-message-bubble--' + 'y']"
)
```

**Comment by LeCoupa** [comment]
Thank you for your answer. That's strange, it was highlighted perfectly last week.

**Comment by stgogm** [comment]
I can confirm this is happening. Pug syntax highlighting works well with `v0.14.5` but with `v0.15.0` is broken.

**`v0.14.5`:**
![image](https://user-images.githubusercontent.com/2774311/52488819-77ba4000-2b9f-11e9-847b-c7257c7c9fb5.png)

**`v0.15.0`**:
![image](https://user-images.githubusercontent.com/2774311/52488884-a0dad080-2b9f-11e9-9ab6-d84872775673.png)

This bug causes all the code inside a Vue component to be wrongly colored. Even the `<script>` tags after it:
![image](https://user-images.githubusercontent.com/2774311/52489005-f2835b00-2b9f-11e9-84d3-50959795da0b.png)

It might be related to the last VSCode update and maybe some change to the Pug syntax highlighter.

**Comment by Patcher56** [comment]
I already created a pull request to fix that problem with multi line strings.. (should be happening with vue-html too)

The thing is, that with the 0.14.5 version there was no real syntax highlighting for vue in pug. (every statement is colored like a string), with 0.15.0 the code inside vue directives (e.g. v-if, :class) and vue-interpolations ({{ }}) is syntax highlighted in javascript.

Anyway with the latest commit in my pull request multi line strings created with the ` character should be formatted right. Until then just use one-liners please.

**Comment by octref** [comment]
This is fixed, thanks to the PR from @Patcher56. I'll publish a version later today.

**Comment by octref** [comment]
Published the change in 0.15.1.
